src/ContentValidationListener.php

```diff
@@ -49,7 +49,7 @@ class ContentValidationListener implements ListenerAggregateInterface, EventMana
     protected $inputFilters = [];
 
     /**
-     * @var CallbackHandler[]
+     * @var callable[]
```

Comment:
Not true in v2 usage, though, which is why I recommend using the `ListenerAggregateTrait`, and removing the property altogether.